todo_app/data/trello_items.py

```diff
@@ -0,0 +1,45 @@
+import requests, os, urllib.parse
+from todo_app.item import Item
+trello_api_key = os.getenv('TRELLO_API_KEY')
+trello_api_token = os.getenv('TRELLO_API_TOKEN')
+trello_board_id = os.getenv('TRELLO_BOARD_ID')
+trello_incomplete_list_id = os.getenv('TRELLO_INCOMPLETE_LIST_ID')
+trello_completed_list_id = os.getenv('TRELLO_COMPLETED_LIST_ID')
+base_url = 'https://api.trello.com/1/'
```

Comment:
This works nicely here, but pre-loading these may bite you in the next module so worth keeping an eye out for it. 

The reason it could be a problem is that these lines of code run when this file is first _imported_. That works perfectly right now, but:
* When we introduce tests we'll sometimes want to do some setup before we start the app
* This can include loading fake/test environment variables
* But these variables will already have had their values assigned by the time we get to our setup, so we won't affect them

We can fix that by messing with our import order carefully, but we typically end up with very fragile code, so a better solution is one of:
* The easiest here, just load the environment variables when they're needed, e.g. in `make_request`:
```python
    query_params = {'key': os.getenv('TRELLO_API_KEY'), 'token': os.getenv('TRELLO_API_TOKEN')}
```
* Personally I'd be lazy and go with the above here, but on a real codebase we'd probably prefer a more classic OOP solution; wrap these functions in a class, and load the environment variables in a constructor, e.g. something like:
```python
class TrelloService():

    def __init__(self):
        self.trello_api_key = os.getenv('TRELLO_API_TOKEN')
        ...
    def make_request(method, endpoint, params=None):
        url = urllib.parse.urljoin(base_url, endpoint)
        query_params = {'key': self.trello_api_key, 'token': self.trello_api_token}
        ...
```

Reply:
Also thanks for the most detailed comment I've ever received on a PR, much appreciated 😄

Reply:
(Given that last comment, I am duty bound to admit that this is a modification of a copy-pasted comment on one of your cohort's reviews who had a very similar set up! 😄)